### PR TITLE
Add logs

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/Conference.java [hunk 1]
```diff
@@ -493,16 +493,12 @@ private void dominantSpeakerChanged()
     {
         Endpoint dominantSpeaker = speechActivity.getDominantEndpoint();
 
-        if (logger.isTraceEnabled())
-        {
-            logger.trace(
-                    "The dominant speaker in conference " + getID()
+        logger.info("The dominant speaker in conference " + getID()
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/Conference.java]: We could potentially keep the if statement to guard against unnecessary string concatenation.

File: src/main/java/org/jitsi/videobridge/simulcast/SimulcastReceiver.java [hunk 2]
```diff
@@ -399,6 +400,12 @@ public void accepted(RawPacket pkt)
         // timestamps of the RTP packets.
         long pktTimestamp = pkt.getTimestamp();
         boolean frameStarted = false;
+        boolean isKeyFrame = isKeyFrame(pkt);
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/simulcast/SimulcastReceiver.java]: We could potentially have this inside an if (INFO) statement.

File: src/main/java/org/jitsi/videobridge/Conference.java (hunk 1 above)
Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Conference.java]: We don't check for INFO level anywhere else. Would you prefer if I add the check here?

Comment (bgrozev) [inline src/main/java/org/jitsi/videobridge/Conference.java]: I stand corrected. Adding the check.

**Pull request merged**